public/fonts/fonts.css

```
@@ -61,11 +61,19 @@
   font-display: swap;
 }
 
-/* Chonburi */
+/* HS Bombaram */
 @font-face {
-  font-family: "Chonburi";
-  src: url("./Chonburi-Regular.woff") format("woff");
-  font-weight: 100 900;
+  font-family: "HSBombaram";
```

Stylelint (17.14.0)

[error] Expected no quotes around "HSBombaram" (font-family-name-quotes)

(font-family-name-quotes)